### PR TITLE
Adds export for embedded_assistant.proto

### PR DESCRIPTION
**Pull request by Fleker**
- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -74,6 +74,14 @@ describe('google-proto-files', function() {
       resolve('privacy/dlp/v2beta1/dlp.proto')
     );
   });
+  it('should export embedded_assistant', function() {
+    console.log(resolve('assistant/embedded/v1alpha2/embedded_assistant.proto'), googleProtoFiles.embedded_assistant.v1alpha2);
```
Comment (stephenplusplus) [inline test/index.test.js]: Please remove the log.

File: index.js [hunk 2]
```diff
@@ -48,6 +48,15 @@ module.exports.dlp = {
   v2beta1: googleProtoFiles('privacy/dlp/v2beta1/dlp.proto'),
 };
 
+module.exports.embedded_assistant = {
```
Comment (stephenplusplus) [inline index.js]: Do you mind casing it `embeddedAssistant` instead to match more conventional JS style? The other names in the repo use all lowercase, e.g. `module.exports.storagetransfer` and `module.exports.videointelligence`. That seems strange to me, but if that's more preferable, feel free.

Comment (Fleker) [reply index.js]: Done

File: test/index.test.js (hunk 1 above)
Comment (Fleker) [reply test/index.test.js]: Done

**Pull request merged**